src/dotnet-toolbox/VersionFile/VersionFile.cs

```
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace DotNetToolbox.VersionMetadata
+{
+    public class VersionFile
+    {
+        private static List<PackageMetadata> _contents = new List<PackageMetadata>();
```

Is there a reason to have this static instance? JSON.NET will create a new instance regardless. Suggest just making this a local variable in the WriteVersion method.

---

There is no reason to have the static instance. But there is a reason to new it up, since I want to silently proceed in the case the file is not there (which can happen on first run).